### DISCUSSION TITLE
Battleground queue announcer #missyou

### DISCUSSION THREAD

**Discussion by creativemonkeyz** [chromiecraft/chromiecraft#3259] [Suggestions]
I don't know why the battleground queue announcer was disabled (probably because of 'spam') but this killed the queue. Now i have to wait like 15-20 min for a WSG (lvl60) queue because nobody know how many ppl are in queue.
I know there is that command that can help you find out how many and on which battleground players are queued in but it doesn't even work properly, i queued as a lvl 60 for wsg and it showed me lvl 10-19 and 20-29 queue.
But i did not opened this topic to criticize the workability of the command...i just want to say that not everybody know about that command and that queue announcer had a great role..and don't tell me this is a big spam...we are max 1k ppl online.
Queue now only announce when bg starts...which is not enough, ppl will know that other players are interested in that bg only when it starts.
Please consider this, it's a big difference (negative difference) since the announcer was disabled for queue.

Thank you

**Comment by 55Honey** [comment]
Hi creativemonkeyz!
The BG queue announcer is still visible to all players by default.
If a player has turned it off via command, they will only see **ALL** BGs that pop. No longer changes to the number of players in the queue. They need to re-enable the queue via the command to see the detailed info again.

**Comment by 55Honey** [comment]
I just learned that the BG queue announcer is temporarily disabled. I'm sorry for the confusion. We will work on bringing it back as soon as possible.

**Comment by creativemonkeyz** [comment]
Hello and thank you for your answer,
I understand, this is great news.
